FirebaseMessaging/Sources/FIRMessaging.m

```diff
@@ -540,7 +544,8 @@ - (void)setAutoInitEnabled:(BOOL)autoInitEnabled {
 
 - (NSString *)FCMToken {
   // Gets the current default token, and requets a new one if it doesn't exist.
-  return [[FIRMessaging messaging].tokenManager tokenAndRequestIfNotExist];
+  NSString *token = [[FIRMessaging messaging].tokenManager tokenAndRequestIfNotExist];
```

Comment:
I wonder why `[[FIRMessaging messaging].tokenManager` instead of `self. tokenManager` ?

Reply:
Must be missed through my refactor, yeah changed to self.tokenManager